scripts/generate_export.py

```diff
@@ -451,11 +455,28 @@ def sequences_overlap(true_seq, pred_seq):
             ].apply(lambda x: x.ffill().bfill())
             output_df = output_df[col_order]
 
+            for cs in complete_submissions:
+                complete_filenames.append(str(cs.input_filename))
+
+            complete_filenames_df = pd.DataFrame(complete_filenames, columns=["filename"])
+        
+            for cs in complete_submissions:
+                sub_job = (IndicoClient.call(SubmissionResult(cs.id, wait=True)))
+                result = (IndicoClient.call(RetrieveStorageObject(sub_job.result)))
+                complete_revID.append(result.get('reviewer_id'))
```

Comment:
If we're just getting the reviewer id, we should add it as an output to get_page_extractions:
https://github.com/IndicoDataSolutions/solutions_toolkit/blob/ca44a1a0e20d1215b4aae314bbb2781603ce75e9/scripts/generate_export.py#L36

Downloading storage objects is an expensive operation and we should try to avoid duplicating the call.